templates/home/home-en.html

```diff
@@ -15,12 +15,12 @@ <h1 property="name" id="wb-cont">Canada.ca</h1>
 			<p class="mrgn-tp-md">The official website of the Government of Canada</p>
 		</div>
 	</div>
-	<div class="bg-gctheme opct-90">
+	<div class="provisional gc-featured-link">
 		<div class="container">
 			<div class="row">
-				<div class="mrgn-tp-md mrgn-bttm-md col-xs-12">
-					<p class="pull-left text-white mrgn-bttm-0">Featured:&nbsp;</p>
-					<p class="pull-left mrgn-bttm-0"><a class="stretched-link text-white" href="#">[Promotion title]</a></p>
+				<div class="col-xs-12">
+					<p>Featured:&nbsp;</p>
+					<p><a class="stretched-link" href="#">[Promotion title]</a></p>
 				</div>
 			</div>
 		</div>
```

Comment:
The old pattern need to be documented somewhere. May be with the gc-featured-link.

Reply:
@duboisp Can we make it as a second working example on the "Documentation and working example" fo now at least it is tracked? ans since it is a provisional component we'll be able to do proper documentation to stabilize it.